### PR TITLE
chore: St. Albans key, chain, and cert

### PR DESCRIPTION
**Pull request by DafyddLlyr**
Take 3...! 🤞 

St. Albans have now provided a chain certificate which should resolve the previous issues. Again this has been tested on AWS staging without issues.

### CODE REVIEW

File: infrastructure/application/index.ts [hunk 1]
```diff
@@ -52,6 +52,10 @@ const CUSTOM_DOMAINS =
           domain: "planningservices.newcastle.gov.uk",
           name: "newcastle",
         },
+        {
+          domain: "stalbans.newcastle.gov.uk",
```
Comment (Mike-Heneghan) [inline infrastructure/application/index.ts:56]: Is this right or a copy and paste error?

Comment (DafyddLlyr) [reply infrastructure/application/index.ts:56]: 🤦

Comment (DafyddLlyr) [reply infrastructure/application/index.ts:56]: Thank you very much! Will fix now

**Review by Mike-Heneghan — CHANGES_REQUESTED** [review]
I think there's maybe a wee copy/paste error for the domain for St Albans.

**Comment by DafyddLlyr** [comment]
Thanks very much for catching that @Mike-Heneghan - clearly time for a walk / coffee / both 😅 Fixed!

**Review by Mike-Heneghan — APPROVED** [review]
Looks good to me 👍 

Non-blocking but in a previous PR I think you included an [Airbrake config change](https://github.com/theopensystemslab/planx-new/pull/2390/files) but I don't think it blocks this.

**Comment by DafyddLlyr** [comment]
That's correct - it's intentionally left out here. That change is already queued up here once the CDN is generated https://github.com/theopensystemslab/planx-new/pull/2382/files

**Pull request merged**